ufc/bandit-tests/test-case-banner-bandit.json

```diff
@@ -211,6 +211,56 @@
             }
         ],
         "assignment": {"variation": "banner_bandit", "action": "nike"}
+      },
+      {
+        "subjectKey": "henry",
+        "subjectAttributes": {
+          "numericAttributes": { "age": 25, "mistake": "oops" },
```

Comment:
`mistake` should be ignored (vs. throwing an error)

Reply:
Great call out yes I think it's warning worthy

Reply:
It's not obvious to me that this is warning-worthy. It would trigger warnings every time a new attribute is used, in particular for _all_ attributes during the cold start phase (when the bandit model is empty or missing). Info-level logging, sure: "hey we found a new attribute". These go away automatically after a new bandit training run, when the attribute will now be part of the model. New attributes are not necessarily indicative of a problem.

Reply:
It's not that they're ignored by the model - it's that they're ignored as input at the SDK layer because they've been passed as the wrong attribute type (a string value listed with the numeric attributes). Strings passed specifically as "numeric" attributes are dropped when model evaluation happens. It means that the developer has made a mistake in classifying subject/action attributes in their own code/data and should fix it.

Reply:
You are very right! I missed the (now obvious) type mismatch. Agree that it should be a warning.